src/chigame/api/tests/test_api.py

```diff
@@ -282,3 +282,26 @@ def test_delete_message(self):
         self.assertEqual(data2["update_on"], Message.objects.get(id=2).update_on)
         self.assertEqual(data2["content"], Message.objects.get(id=2).content)
         self.assertEqual(2, Message.objects.get(id=2).token_id)
+
+
+class UserTests(APITestCase):
+    def test_user_get(self):
+        user = UserFactory()
+
+        list_url = reverse("api-user-list")
+        detail_url = reverse("api-user-detail", kwargs={"pk": user.id})
+
+        list_response = self.client.get(list_url)
+        assert list_response.status_code == 200
+
+        detail_response = self.client.get(detail_url)
+        assert detail_response.status_code == 200
+
+    def test_user_delete(self):
+        user = UserFactory()
+
+        url = reverse("api-user-detail", args=[user.id])
+        response = self.client.delete(url, format="json")
+
+        self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
```

Comment:
This looks good to me! Perhaps in this case, have an assert to ensure that a user was made in the first place and then the count goes to 0 after its deletion!